src/main/java/lembas/core/api/common/SimpleLogger.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (C) 2020 Lembas Modding Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package lembas.core.api.common;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+/**
+ * A simple wrapper for a logger using String.format for formatting the log

```

Comment:
Using `String.format` is not a necessity, and should thus not be stated as such here.

Reply:
What I mean there is that the logger uses `String.format` - this is an requirement for the implementation. The reason why it's here is that the argument syntax is interpreted that way (the formatting specifiers for `String.format` are used) - of course I could make that a provider specific thing, but this bypasses the idea of a really simple logger class and would make the logger creation more complicated.

Reply:
In that case I suggest
```suggestion
 * A simple wrapper for a logger that should use String.format for formatting the log
```